app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -166,7 +166,7 @@ def rbac_tenant_manage_quotas_reset
   end
 
   def rbac_tenant_manage_quotas
-    assert_privileges("rbac_tenant_manage_quotas")
+    assert_privileges(MiqProductFeature.tenant_identifier('rbac_tenant_manage_quotas', params[:id]))
```

Comment:
Would be better if the call to `MiqProductFeature.tenant_identifier` lived in `assert_privileges`, if possible. This way UI developers don't necessarily need to be aware of it.